Update currencies-overview.md (#575)
bigcommerce/dev-docs: docs/api-docs/catalog/currencies/currencies-overview.md

### On this Page

> - [Display vs. Transactional](#display-vs-transactional)

Reviewer: ```suggestion
- [Display vs. transactional](#display-vs-transactional)

```
Should be updated in article as well

<!-- theme: info -->

### Note

> > * If the shopper's card-issuing bank differs from the merchant's bank account country, additional cross-border fees may apply that can change the final settlement amount.

Reviewer: ```suggestion
> If the shopper's card-issuing bank differs from the merchant's bank account country, additional cross-border fees may apply that can change the final settlement amount.

```

> * You can choose whether a currency should be display-only or transactional when adding or editing a currency.

</div>
</div>
</div>

> ## Pre-configuring the Store

Reviewer: ```suggestion
## Preconfiguring the store

```

<a id="pre-configuring-the-store"></a>

> Before setting up multiple currencies on a store, ensure to configure the store as follows:

Reviewer: ```suggestion
Before setting up multiple currencies on a store, be sure to configure the store as follows:

```

* Stripe Payment Gateway
* Stencil Theme
* Optimized One-Page Checkout

Additionally, review the [Currency API reference](https://developer.bigcommerce.com/api-reference/store-management/currency-api) to get acquainted with the endpoints and resources related to configuring multiple currencies.

<div class="HubBlock--callout">
<div class="CalloutBlock--info">
<div class="HubBlock-content">

<!-- theme: info -->

> > Make sure not to change the store’s default currency. This change could cause currency to not function properly.

Reviewer: ```suggestion
> Do not change the store’s default currency. This change could cause currency to not function properly.

```

> > Make sure not to change the store’s default currency. This change could cause currency to not function properly.

Reviewer: What does "This change could cause currency to not function properly..." mean?
This sentence needs some clarification.

</div>
</div>

## How currencies work

<a id="how-currencies-work"></a>

For details on how to surface currencies throughout BigCommerce APIs, user interfaces, and storefront components, see [How Currencies Works](https://developer.bigcommerce.com/api-docs/catalog/currencies/how-currencies-work).

## Supported features

<a id="supported-features"></a>

> See the table below for a list of supported and unsupported features.

Reviewer: We may need to rename the header for this section. It's a bit misleading to only mention supported features when we also list the unsupported ones

> | Name | Description |

Reviewer: The header for this section needs a more descriptive name; definitions of what?

> | Name | Description |

Reviewer: The definitions of these features don't need to restate the names in the descriptions

| Dynamic Conversion Rate | One of the two auto-converted pricing options. If a merchant selects a dynamic conversion rate, they've tied themselves to a currency conversion service, which will update the conversion rate at a certain frequency. This option helps shopper-facing pricing remain most aligned to the store's default currency and keeps non-default currency conversion rate at market rate. Merchants can use BigCommerce Currency Service provided in the Currency setup page, or they can use API to update the exchange rate automatically from their trusted source. |
| Bank Conversion Rate | Conversion rate is used by merchants' or shoppers' payment or credit card providers when auto-converting from store's transactional currency. This rate might not align with the BC conversion rate, and BC has no visibility into it. |

> | Multi Currency Pricing | Rather than opting for auto-converting product pricing from default currency using BC conversion rate, the merchant has a choice to set price per product per currency. You can implement Multi Currency Pricing through price lists. |

Reviewer: ```suggestion
| Multi-Currency Pricing | Rather than opting for auto-converting product pricing from default currency using BC conversion rate, the merchant has a choice to set price per product per currency. You can implement Multi-Currency Pricing through price lists. |

```

> ## FAQ

Reviewer: All answers to the bolded questions should be on separate lines

<a id="faq"></a>

**Do Multi-Currency settings work with the Checkout SDK?**
The Checkout SDK works with multi-currency. There is no additional setup needed for the SDK.

> After setting up the currency in the Control Panel, the SDK will work as normal.

Reviewer: ```suggestion
After setting up the currency in the control panel, the SDK will work as normal.

```

**Do my theme customizations work with Multi-Currency Settings?**

> If you set up the theme to show the currency on the storefront, there should be no issues. Please see your

Reviewer: ```suggestion
If you set up the theme to show the currency on the storefront, there should be no issues. Please refer to your

```
